packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -544,3 +584,47 @@ class AndroidViewController {
     _state = _AndroidViewState.created;
   }
 }
+
+/// Controls an iOS UIView.
+///
+/// Typically created with [PlatformViewsService.initUiKitView].
+class UiKitViewController {
+  UiKitViewController._(
+    this.id,
+    TextDirection layoutDirection,
+  ) : assert(id != null),
+      assert(layoutDirection != null),
+      _layoutDirection = layoutDirection;
+
+
+  /// The unique identifier of the iOS view controlled by this controller.
+  final int id;
+
+  bool _disposed = false;
```

Comment:
i would call this _debugDisposed and only track it in debug mode

Reply:
done